app/lib/primer/classify.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require_relative "classify/cache"
```

Comment:
Tests are passing so I assume it's fine, but is this already included elsewhere?

Reply:
Since I moved the helpers from `lib` to `app/lib`, the engine now requires everything by itself (once), so we don't have to worry about manually including files